packages/eslint-plugin-pf-codemods/lib/rules/v5/clipboardCopy-remove-popoverPosition.js

```diff
@@ -0,0 +1,45 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8226
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const clipboardImports = getPackageImports(
+      context,
+      "@patternfly/react-core"
+    ).filter((specifier) =>
+      ["ClipboardCopy", "ClipboardCopyButton"].includes(specifier.imported.name)
+    );
+
+    return clipboardImports.length === 0
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            if (
+              clipboardImports
+                .map((imp) => imp.local.name)
+                .includes(node.name.name)
+            ) {
+              const positionAttr = node.attributes.find(
+                (nodeAttr) => nodeAttr.name && nodeAttr.name.name === "position"
+              );
+              if (
+                positionAttr.value.expression &&
+                positionAttr.value.expression.object.name === "PopoverPosition"
```

Comment:
`positionAttr` could be undefined at this point which would cause an error, think you could just..
```suggestion
                positionAttr?.value?.expression?.object?.name === "PopoverPosition"
```

Reply:
can test this with a `valid` test like:

    code: `import { ClipboardCopy } from '@patternfly/react-core'; <ClipboardCopy />`